index.js

```diff
@@ -180,14 +178,29 @@ function addRoutesOnModel( routes, urlPrefix, routeName, model ) {
 	function reqListMatches( req, res ) {
 		Log( "got request listing matching items" );
 
-		const { offset = 0, limit = Infinity } = req.query;
+		const { offset = 0, limit = Infinity, sortBy = "", descending = false } = req.query;
 		const { attribute, value, operator } = req.params;
 		const meta = req.headers["x-count"] ? {} : null;
 
-		return model.findByAttribute( attribute, value, operator, offset, limit, meta )
+		const sortModification = descending? -1: 1;
+
+		return model.findByAttribute( attribute, value, operator, 0, Infinity, meta )
 			.then( matches => {
 				const result = {
-					items: matches.map( match => match.toObject() ),
+					items: matches.map( match => match.toObject() ).sort(( l , r ) => {
+						const latt = l[sortBy];
+						const ratt = r[sortBy];
+						if( latt == null && ratt == null ){
+							return 0;
+						}
+						if( latt == null ){
+							return sortModification;
+						}
+						if( ratt == null){
+							return -1 * sortModification;
```

Comment:
This looks wrong in case records with attribute value `null` should always appear at end of sorted list.

Reply:
Fixed in latest commit